Workiva/dart_transformer_utils
# UIP-811: RELEASE dart_transformer_utils 0.1.1

__This release contains commits I felt didn't warrant their own CRs.__

List of commits and testing instructions:

*  https://github.com/Workiva/dart_transformer_utils/commit/61fac6d0145bb84bbd45e27c7a2cbae3ccbd594e Add SDK version constraint to enable carets in dependencies 
    * Pull this branch
    * Verify that the package can be published to Pub without any issues:
        * Run `pub publish --dry-run`.
        * Verify that the output reads `Package has 0 warnings.`.
* https://github.com/Workiva/dart_transformer_utils/commit/c6bcb723507f4d8ae2c10e023e53509189cde990 Set up Travis, remove Smithy config
    * Verify that the Travis CI build is passing.
    * Verify that the  link on the new badge in the README goes to the right place (the badge itself won't work until this is merged to master).

---

## Changes
* Add SDK version constraint in `pubspec.yaml` to enable carets in dependencies.
    * Necessary to eliminate warning that would have prevented Pub publishing.
* Set up Travis CI.

---

Please review: @evanweible-wf @jacehensley-wf @clairesarsam-wf @joelleibow-wf
